sphinx/tutorial/rose/suites.rst

```diff
@@ -476,9 +490,6 @@ Otherwise an add-hoc web-server can be set up using the
       * ``WIND_FILE_TEMPLATE``
       * ``RAINFALL_FILE``
       * ``MAP_FILE``
-      * ``CYLC_TASK_CYCLE_POINT``
-      * ``RESOLUTION``
-      * ``DOMAIN``
```

Comment:
This was right before!

Reply:
Indeed. I rushed through this and kept making mistakes that led me to think there was issue, but the practical was actually fine!